tests/framework/extensions/provisioninginput/config.go

```diff
@@ -172,4 +172,5 @@ type Config struct {
 	UpgradeStrategy        *rkev1.ClusterUpgradeStrategy            `json:"upgradeStrategy" yaml:"upgradeStrategy"`
 	Advanced               *Advanced                                `json:"advanced" yaml:"advanced"`
 	ClusterSSHTests        []string                                 `json:"clusterSSHTests" yaml:"clusterSSHTests"`
+	DrainBeforeDelete      bool                                     `json:"drainBeforeDelete" yaml:"drainBeforeDelete"`
```

Comment:
With looking how the UI is, this should go in the different machine configs instead. This a machine pool configuration, and treating it like this doesn't make sense. That way in the near future when we update our machine configs to be multi-pool configuration it would make much more sense.

Reply:
That's a good point, I agree w/Israel. This is a short-term solution that will cause a long-term issue. So it's better to just handle it now.

Reply:
Spoke with Caleb about this, and we decided that it would be best if this went in a nodeRoles as new var. I'm not sure if he mentioned it to the team.

Reply:
Sorry I forgot to remove this